velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -399,6 +399,21 @@ TEST_F(ArithmeticTest, radians) {
   EXPECT_DOUBLE_EQ(-1.0000736613927508, radians(-57.3).value());
 }
 
+TEST_F(ArithmeticTest, degrees) {
+  const auto degrees = [&](std::optional<double> a) {
+    return evaluateOnce<double>("degrees(c0)", a);
+  };
+
+  EXPECT_EQ(std::nullopt, degrees(std::nullopt));
```

Comment:
Please add tests for values std::numeric_limits<double>::min(), max(), infinity(), quiet_NaN()